assets/sass/_navigation.scss

```diff
@@ -216,47 +216,47 @@ Style guide: Navigation.3 Skip Links
   }
 }
 
-// Document me please! :)
-//
-// Card ID for the doc work: DESIGN-153
-article {
-  header {
-    margin-bottom: $base-spacing * 2;
-
-    // Jump to link heading, inside the article header
-    h2 {
-      @extend h6;
-
-      margin: ($base-spacing * 2) 0 0 $small-spacing;
-      padding: 0 0 $tiny-spacing ($base-spacing - $tiny-spacing);
-      border-left: $tiny-spacing solid $light-grey;
-      color: $grey;
-      font-weight: $base-font-weight;
-    }
 
-    .jump-links {
-      margin: 0 0 0 $small-spacing;
-      padding: $medium-spacing 0 $medium-spacing ($base-spacing - $tiny-spacing);
-      border-left: $tiny-spacing solid $light-grey;
+/*
+Index links
 
-      @include media ($tablet) {
-        column-count: 2;
-        column-width: 40%;
-        column-gap: $gutter * 4;
-      }
+Used to group a collection of text links, Index links can appear at the beginning of an article page as a table of contents, or as a group of links to related content.
 
-      li {
-        margin-bottom: $medium-spacing;
-        font-weight: $bold-font-weight;
+An optional heading is included in the markup example. Any level leading from `h2` to `h6` can be used. Use a heading level that's appropriate for your document outline.
```

Comment:
I like this, a lot. Nice addition — I saw the code that reflected this and was confused at first, but it makes total sense.